.github/TESTING.md

```diff
@@ -57,23 +99,37 @@ Before a release doil needs to be tested deeply. For that we implemented a manda
 * [ ] it's possible to login via cli
 * [ ] the available states can be listed
 
+### Managing the mail server
+
+* [ ] the server can be started
+* [ ] the server can be stopped
+* [ ] the server can be restarted
+* [ ] it's possible to login via cli
+* [ ] the password can be changed
+
 ### Additional functionalities
 
-* [ ] Currently installed instances can be listed
+* [ ] Currently, installed instances can be listed
```

Comment:
The comma is not correct here, as we are refering to instances that are currently installed.